docs/guides/admin/docs/modules/metrics.md

```diff
@@ -51,6 +52,10 @@ requests_total 1.0
 opencast_asset_manager_events{organization="mh_default_org",} 1.0
 ```
 
+Additionally, standard JVM metrics are exported providing information about e.g. memory and CPU usage, threads,
+classloading, etc. The JVM metrics collectors are documented [here](https://prometheus.github.io/client_java/).
+A corresponding monitoring mixin with dashboards and alerting rules can be found
+[here](https://github.com/grafana/jsonnet-libs/tree/master/jvm-mixin).
```

Comment:
It's always nice to see where you go instead of having a generic “click here” link:

```suggestion
Additionally, standard JVM metrics are exported providing information about e.g. memory and CPU usage, threads,
classloading, etc. For more information, take a look at the [Java client's JavaDoc](https://prometheus.github.io/client_java/).
A corresponding monitoring mixin with dashboards and alerting rules can be found at
[github.com/grafana/jsonnet-libs](https://github.com/grafana/jsonnet-libs/tree/master/jvm-mixin).
```

Though I don't think the JavaDocs are actually that helpful for end-users and it would be nice to link a metrics descriptions instead.